packages/lib/testMountParameters/README.md

```diff
@@ -1,4 +1,4 @@
 # createTestMountParameters
 
-This is a helper function for vue testing. It returns fresh `MockParameters`, including a `localVue`, `vuetify`, and `vuex` instance, the latter copying the basic `core` store structure, i.e., having an empty namespaced `plugin` module.  
```

Comment:
This whitespace should stay, as it leads to a new line.

Reply:
Restored the whitespace :)